pkg/utils/printer/printer.go

```diff
@@ -99,6 +101,32 @@ func (p *JSONPrinter) Print(v interface{}) error {
 	return nil
 }
 
+func (p *JSONPrinter) Infof(format string, a ...interface{}) {
```

Comment:
Would it make more sense to name these functions `Verbose*` ?

Reply:
since it's used only for the verbose flag, yes, I think it makes sense